.bmad/bmm/docs/party-mode.md

```
@@ -0,0 +1,224 @@
+# Party Mode: Multi-Agent Collaboration
+
+**Get all your AI agents in one conversation**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)